module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,17 @@
+#include "vowel.hpp"
+
+#include <algorithm>
+#include <array>
+
+void removeVowels(std::vector<std::string>& vec) {
+    std::array<char, 12>
+        vowels{'a', 'e', 'i', 'o', 'u', 'y', 'A', 'E', 'I', 'O', 'U', 'Y'};
```

Comment:
I think you should make it constrexpr and put it outside of remoweVovels method. It would make easier to change what vowels you are removing in future if your program would be larger.

Reply:
We made branch from our previous exercise old solution, so this stuff was corrected. Accidentaly we included to STL#2 PR for files from STL#1.

Reply:
I don't agree to remove array of vovels outside of the metod for purpose of this program, we don't use it anywhere else